medyator/kink/kink_service_provider.py

```
@@ -1,25 +1,55 @@
-from typing import Union, cast
+from typing import Union, cast, Any
 
 from kink import Container
 
-from ..contracts import BaseRequest, Command, Query, ServiceProvider
+from ..contracts import (
+    AsyncCommand,
```

Ruff (0.11.9)

`..contracts.AsyncCommand` imported but unused

Remove unused import

(F401)